kolibri/plugins/learn/assets/src/views/action-bar-search-box/index.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+
+  <div class="search-box-wrapper">
+    <div v-show="showSearchBox" :class="{ 'search-box-dropdown': showDropdownSearchBox }">
```

Comment:
From vuejs documentation on [v-if vs v-show](https://vuejs.org/v2/guide/conditional.html#v-if-vs-v-show): 
> Generally speaking, v-if has higher toggle costs while v-show has higher initial render costs. So prefer v-show if you need to toggle something very often, and prefer v-if if the condition is unlikely to change at runtime.

Here the condition, aka the size of window, is likely to change at runtime, so I use v-show.